### ISSUE TITLE
Error: No provider for: jupyter.services.commandpalette

### ISSUE THREAD

**Issue by rornor** [dask/dask-labextension#6]
Hi,

I use latest jupyter lab (0.4.1) with Python 3.5 on Windows.
I installed this extension as in readme instructions, but when I launch jupyter lab, I get blank screen.

Console shows this:

```
Uncaught (in promise) Error: No provider for: jupyter.services.commandpalette.(…)
    Application.resolveService @ main.bundle.js:276
    (anonymous function) @ main.bundle.js:238
    Application.activatePlugin @ main.bundle.js:238
    (anonymous function) @ main.bundle.js:324
    Application.start @ main.bundle.js:324
    (anonymous function) @ lab:57
```

When I breakpoint main.bundle.js:276, I see `token = Token {_name: "jupyter.services.commandpalette"}` while `id` is undefined.

If I uninstall this extension everything is back to normal, and jupyter lab works as expected.

**Comment by mrocklin** [comment]
This library is experimental not yet for public use.

**Comment by blink1073** [comment]
Hi @rornor, this is a known issue with JupyterLab itself (growing pains): https://github.com/jupyter/jupyterlab/issues/1011

**Comment by canavandl** [comment]
resolved by #7
